strawtubes/strawtubes.cxx

```diff
@@ -1,7 +1,7 @@
 // First version of the strawtracker geometry, based on NA62 straws
 // 7/10/2015
 // E. van Herwijnen eric.van.herwijnen@cern.ch
-// Also contains (for the moment) the veto station
+// Veto station removed by W.-C. Marty Lee
```

Comment:
This info is already contained in the commit history and changelog, no need to repeat it here.